projects/extension/sql/idempotent/008-embedding.sql

```diff
@@ -21,6 +21,32 @@ $func$ language sql immutable security invoker
 set search_path to pg_catalog, pg_temp
 ;
 
+-------------------------------------------------------------------------------
+-- embedding_ollama
+create or replace function ai.embedding_ollama
+( model text
+, dimensions int
+, base_url text default null
+, truncate boolean default true
+, options jsonb default null
+, keep_alive text default null
```

Comment:
I just saw that in the `ai.ollama_embed` function, we use `float8` for `keep_alive`: https://github.com/timescale/pgai/blob/eb697883c30634839d1fc4e4b3efb98d59d5283d/projects/extension/sql/idempotent/002-ollama.sql#L102. The python client supports both float8 and text. I chose text because I feel like duration specifiers like `10m` are easier to use than `600.0`. I wonder what we should do about aligning these. @jgpruitt WDYT?

Reply:
I'm ambivalent. I'm okay with your aligning them if you want.

Reply:
I considered adding a second overload with `keep_alive` having type `numeric`, but that doesn't work because then postgres can't determine which overload to use if you don't pass the optional parameter (e.g. `ai.embedding_ollama('test-model', 1234);`).

I'm going to stick to this API, as it's the more general one (you can also call it with `keep_alive => 600.0::text`).